README.md

```diff
@@ -36,6 +36,8 @@ Repo tokens are necessary to distinguish your repository from others. You can fi
 export CODECOV_TOKEN=":uuid-repo-token"
 # or
 ./node_modules/.bin/codecov --token=:token
+# or
+nyc report --reporter=text-lcov | ./node_modules/.bin/codecov --pipe
```

Comment:
Should be:

```
./node_modules/.bin/nyc report --reporter=text-lcov | ./node_modules/.bin/codecov --pipe
```

Reply:
👍

Reply:
Hmm.. that is correct if you invoke codecov from the terminal. But if you set it in `package.json` script section, then you should not specify the path. But since all of the examples are written as invoked from the terminal, I will change it.